Iterable-iOS-SDK/IterableAttributionInfo.m

```diff
@@ -0,0 +1,40 @@
+//
+//  IterableAttributionInfo.m
+//  Iterable-iOS-SDK
+//
+//  Created by Victor Babenko on 5/24/18.
+//  Copyright © 2018 Iterable. All rights reserved.
+//
+
+#import "IterableAttributionInfo.h"
+
+@implementation IterableAttributionInfo
+
+- (void)encodeWithCoder:(NSCoder *)aCoder {
```

Comment:
Is this going to work and deserialize properly if we change IterableAttributionInfo fields in the future?

Reply:
We should be OK for adding new fields etc. Here is a link
https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/Archiving/Articles/compatibility.html#//apple_ref/doc/uid/20001055-BCICFFGE